x-pack/filebeat/module/zeek/connection/test/connection-json.log-expected.json

```diff
@@ -34,8 +34,8 @@
         "zeek.connection.local_orig": true,
         "zeek.connection.local_resp": true,
         "zeek.connection.missed_bytes": 0,
-        "zeek.connection.state.code": "SF",
-        "zeek.connection.state.msg": "Normal establishment and termination.",
+        "zeek.connection.state": "Normal establishment and termination.",
```

Comment:
This is still a breaking change compared to v7.5. In 7.5 we have `zeek.connection.state: SF` ([example](https://github.com/elastic/beats/blob/v7.5.1/x-pack/filebeat/module/zeek/connection/test/connection-json.log-expected.json#L35)). So to avoid changing the value of that field I would map `zeek.connection.conn_state` to `zeek.connection.state` (as in 7.5). Then add the new field as `zeek.connection.state_message`.

Reply:
If we do it this way, then we will need another PR to change the fields again for 8.0 and above, because we want to use these fields going forward: connection.state and connection.state_code.

Reply:
> because we want to use these fields going forward: connection.state and connection.state_code

Why do we want to use those particular field names going forward?

Reply:
state_code and state seem to make more sense than state and state_message, similar to http status_code, etc.

Reply:
Personally I'm OK with any of the names you mentioned and also how they stand now in master (`state.code` and `state.message`). So perhaps we should just target this PR at the 7.x branch and do a different change for master.

Reply:
After thinking about this more, I agree with you that using the fields `zeek.connection.state` and `zeek.connection.state_message` for both 7.x and master will be cleaner, since there will be no breaking changes needed if we stick to these fields when we move to 8.0. I will go ahead and make the changes accordingly.